README.adoc

```diff
@@ -54,6 +54,17 @@ POST /authenticate
 }
 ----
 
+----
+400 Bad Request
+----
+[source,json]
+----
+{
+    "code": 1,
+    "message": "Unknown user bob"
```

Comment:
Agreed. I just kept the previous message, but liked Stormpath's version more: "Invalid username or password"